Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
```

Comment:
I think the CI error is because the CI actually runs Make as:

    make deps clobber-deps ci-clean fast test

The `$(shell ...)` is expanded at the start, when parsing the makefile. Later on, `ci-clean` deletes some files from under `meteor-testapp`, including files which the `$(shell)` found earlier, hence those files are still listed as dependencies. So then, when Make gets around to building `meteor-testapp.spk`, it thinks these dependencies are missing.

Unfortunately I think you are going to need to list source files more explicitly here. You don't have to list everything, and you can use wildcards. Maybe something like:

    meteor-testapp/client/* meteor-testapp/server/* meteor-testapp/*.js meteor-testapp/.meteor/*